task-ballerina/tests/task_test.bal

```diff
@@ -28,8 +28,8 @@ function getCount() returns int {
     return count;
 }
 
-service appointmentService on appointment {
-    resource function onTrigger() {
+service /appointmentService on appointment {
+    resource function get onTrigger() {
```

Comment:
```suggestion
    remote function onTrigger() {
```

Reply:
Shall we change it to `remote function` instead of `resource function`

Reply:
Updated.